src/js/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -16,6 +16,13 @@ class GuidancePage extends React.Component {
 
   componentDidMount() {
     window.scrollTo(0, 0);
+    localStorage.setItem('dw-viewed-guidance', true);
+
+    if (sessionStorage.getItem('dw-session-started')) {
+      sessionStorage.removeItem('dw-session-started');
```

Comment:
The sessionStorage key `dw-session-started` doesn't seem to be added anywhere in this change set. Not sure if it has been added in previous changes, but check to make sure.